### PR TITLE
i18n(fr): Add `deploy/aws` in French language

### PR DESCRIPTION
**Pull request by thomasbnt**
#### Description (required)

Add `deploy/aws` in French language.

I am participating in the Hacktoberfest! 🎃

- [i18n(fr): Add deploy/aws in French language](https://github.com/withastro/docs/commit/af8bd5de0c4665478589d35b8c55dbe2eceb85d2)
#### Related issues & labels (optional)

Part of [this issue](https://github.com/withastro/docs/issues/4984), don't close it.

### CODE REVIEW

File: src/content/docs/fr/guides/deploy/aws.mdx [hunk 1]
```diff
@@ -0,0 +1,257 @@
+---
+title: Déployez votre site Astro sur AWS
+description: Comment déployer votre site Astro sur le web en utilisant AWS.
+type: deploy
+i18nReady: true
+---
+
+[AWS](https://aws.amazon.com/) est une plateforme d'hébergement d'applications web complète qui peut être utilisée pour déployer un site Astro.
+
+Pour déployer votre projet sur AWS, vous devez utiliser la [console AWS](https://aws.amazon.com/console/). (La plupart de ces actions peuvent également être effectuées à l'aide de [AWS CLI](https://aws.amazon.com/cli/)). Ce guide vous guidera à travers les étapes pour déployer votre site sur AWS en commençant par la méthode la plus basique. Ensuite, il vous montrera comment ajouter des services supplémentaires pour améliorer la rentabilité et les performances.
```
Comment (pioupia) [inline src/content/docs/fr/guides/deploy/aws.mdx:10]: "Ce guide vous guidera" Hum, en effet ! Peut-être ceci pour éviter les répétitions :
```suggestion
Pour déployer votre projet sur AWS, vous devez utiliser la [console AWS](https://aws.amazon.com/console/). (La plupart de ces actions peuvent également être effectuées à l'aide de [AWS CLI](https://aws.amazon.com/cli/)). Ce guide vous accompagnera dans le processus de déploiement de votre site sur AWS, en commençant par la méthode la plus basique. Ensuite, il vous montrera comment ajouter des services supplémentaires pour améliorer la rentabilité et les performances.
```

File: src/content/docs/fr/guides/deploy/aws.mdx [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+
+S3 est le point de départ de toute application. C'est là que sont stockés les fichiers de votre projet et d'autres ressources. S3 facture le stockage des fichiers et le nombre de requêtes. Vous trouverez plus d'informations sur S3 dans la [documentation AWS](https://aws.amazon.com/s3/).
+
+1. Créez un panier S3 avec le nom de votre projet.
+
+:::tip
+Le nom du bucket doit être globalement unique. Nous recommandons une combinaison du nom de votre projet et du nom de domaine de votre site.
+:::
+
+2. Désactivez **"Block all public access "**. Par défaut, AWS définit tous les buckets comme étant privés. Pour le rendre public, vous devez décocher la case "Bloquer l'accès public" dans les propriétés du bucket.
```
Comment (pioupia) [inline src/content/docs/fr/guides/deploy/aws.mdx:75]:
```suggestion
2. Désactivez **"Block all public access"**. Par défaut, AWS définit tous les buckets comme étant privés. Pour le rendre public, vous devez décocher la case "Bloquer l'accès public" dans les propriétés du bucket.
```

File: src/content/docs/fr/guides/deploy/aws.mdx [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+ }
+    ```
+
+:::caution
+N'oubliez pas de remplacer `<DISTRIBUTION_ARN>` et `<BUCKET_NAME>`. Vous pouvez trouver l'ARN dans **CloudFront > Distributions > Details**.
+:::
+
+2. Créez un nouvel utilisateur IAM et attachez la politique à l'utilisateur. Ceci fournira votre `AWS_SECRET_ACCESS_KEY` et `AWS_ACCESS_KEY_ID`.
+
+3. Add this sample workflow to your repository at `.github/workflows/deploy.yml` and push it to GitHub. You will need to add `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, `BUCKET_ID`, and `DISTRIBUTION_ID` as “secrets” to your repository on GitHub under **Settings** > **Secrets** > **Actions**. Click <kbd>New repository secret</kbd> to add each one.
```
Comment (pioupia) [inline src/content/docs/fr/guides/deploy/aws.mdx:223]: Roh trop dommage c'est encore en Anglais !

Comment (thomasbnt) [reply src/content/docs/fr/guides/deploy/aws.mdx]: C'est modifié @pioupia !

**Review by pioupia — APPROVED** [review]
LGTM!

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Great work 💜

**Pull request merged**